src/lib/stores/drips/utils/streamedBetween.ts

```diff
@@ -1,11 +1,68 @@
-import type { EnrichedWorkstream } from '$lib/stores/workstreams';
+import type {
+  DripHistoryEvent,
+  EnrichedWorkstream
+} from '$lib/stores/workstreams';
 import { Currency, type Money } from '$lib/stores/workstreams/types';
+import { getUnixTime } from '$lib/utils/time';
 
 interface TimeWindow {
   from: Date;
   to: Date;
 }
 
+/**
+ * Returns the amount of money earned and spent between the given time window
+ * based on a given history of DripHistoryEvents.
+ */
+export function flattenDripHistory(
```

Comment:
How about `calculateAmountsInTimeframe`?

Reply:
I think that `flatten` is a pretty fitting description of what this function does, tbh. It takes an array of history items and outputs a flat representation of what happened. I don't feel strongly about this at all though. If you do, happy to change :)

Reply:
When I read the name for the first time, I didn't get what it was about, had to read the code to understand. If the name doesn't tell me what it does, I usually look at the return type, hence the suggestion.

 As with everything else, it's just a suggestion, since you're the main dev on this project, i'll leave it up to you to decide :P